Assets/StreamingAssets/LUA/Furniture.lua

```diff
@@ -40,12 +40,19 @@ function OnUpdate_Door( furniture, deltaTime )
 		furniture.ChangeParameter("openness", deltaTime * -4)
 	end
 
-
 	furniture.SetParameter("openness", Clamp01(furniture.GetParameter("openness")) )
 
 	furniture.UpdateOnChanged(furniture);
 end
 
+function OnUpdate_Leak_Door( furniture, deltaTime )
+	furniture.tile.EqualiseGas(deltaTime * 10.0 * (furniture.GetParameter("openness") + 0.1))
```

Comment:
Yes, I might look at doing that tomorrow. I this PR gets merged before that I will do it as part of my next changed to the atmosphere system(Unless someone beats me to it)